pkg/get/get_test.go

```diff
@@ -1237,6 +1237,64 @@ func Test_DownloadK9s(t *testing.T) {
 	}
 }
 
+func Test_DownloadPopeye(t *testing.T) {
+	tools := MakeTools()
+	name := "popeye"
+
+	tool := getTool(name, tools)
+
+	const toolVersion = "v0.21.2"
+
+	tests := []test{
+		{
+			os:      "ming",
+			arch:    arch64bit,
+			version: toolVersion,
+			url:     "https://github.com/derailed/popeye/releases/download/" + toolVersion + "/popeye_Windows_amd64.tar.gz",
```

Comment:
I like to keep the URLs as constant strings where possible, but I get why you did it this way.